configs/jsactions/rollup.config.js

```diff
@@ -0,0 +1,119 @@
+import { red, yellow } from "colors";
+import { mkdirSync, promises } from "fs";
+import { join, relative } from "path";
+import clear from "rollup-plugin-clear";
+import command from "rollup-plugin-command";
+import { cp } from "shelljs";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import { collectDependencies } from "../../packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies";
+import { copyClientDependencies } from "./rollup-plugin-copy-client-dependencies";
+
+const cwd = process.cwd();
+
+export default async args => {
+    const jsActionTargetFolder = `javascriptsource/${args.configProject ?? "nativemobileresources"}/actions`;
+    const result = [];
+    const files = await findFiles();
+    const outDir = join(cwd, "dist");
+
+    files.forEach((file, i) => {
+        const fileInput = file.path.replace(join(cwd, "/"), "");
+        const [fileOutput] = file.name.split(".");
+        result.push({
+            input: fileInput,
+            output: {
+                format: "es",
+                file: join(outDir, `${fileOutput}.js`),
+                sourcemap: false
+            },
+            external: nativeExternal,
+            plugins: [
+                ...(i === 0 ? [clear({ targets: [outDir] })] : []),
+                collectDependencies(nativeExternal, outDir, true, fileOutput, i === 0, true),
+                ...(i === files.length - 1 ? [copyClientDependencies(clientDependencies, outDir)] : []), // Copy extra dependencies not listed in lib's package.json
```

Comment:
Can't we replace this custom plugin with a `command`. Or even put inside the `command` you already have below

Reply:
Good one, just moved